WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/BackupDownloadActivity.kt

```diff
@@ -0,0 +1,64 @@
+package org.wordpress.android.ui.jetpack.backup
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.ViewModelProviders
+import kotlinx.android.synthetic.main.domain_suggestions_activity.*
```

Comment:
Warning (⚠️ ): This seems to be a wrong import. I figured the correct one is this: `import kotlinx.android.synthetic.main.backup_download_activity.*`

PS: Kotlin synthetics, what can you do, right! 🤸‍♀️

Reply:
Oh yeah, nice catch !